src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -61,6 +78,21 @@ def __init__(self, job_id, initial_params, area, should_export_plots):
                 ConstSettings.GeneralSettings.EXPORT_ENERGY_TRADE_PROFILE_HR:
             self.offer_bid_trade_hr = OfferBidTradeGraphStats()
 
+        if constants.KAFKA_MOCK:
+            self.producer = Mock()
+        elif BOOTSTRAP_SERVERS is not None:
+            kwargs = {'bootstrap_servers': BOOTSTRAP_SERVERS,
+                      'sasl_plain_username': SASL_PLAIN_USERNAME,
+                      'sasl_plain_password': SASL_PLAIN_PASSWORD,
+                      'security_protocol': SECURITY_PROTOCOL,
+                      'ssl_context': context, 'sasl_mechanism': SASL_MECHANISM,
+                      'api_version': (0, 10), 'retries': 5, 'buffer_memory': 2048000000,
+                      'max_request_size': 2048000000}
+            self.producer = KafkaProducer(**kwargs)
+        else:
+            kwargs = {'bootstrap_servers': 'localhost:9092'}
```

Comment:
I think it would be good to have this value as an environment value (ideally defined in an `.env` file or something similar)